js/irma-backend/CHANGELOG.md

```diff
@@ -28,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Initial release
 
+[0.1.5]: https://github.com/privacybydesign/irma-backend-packages/compare/6746eac...a478de3
```

Comment:
Let's first merge the fix PR and then check whether the git commit is still correct.

Reply:
It should and it is:)